addons/ai_apps/static/src/message_patch.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="mail.Message" t-inherit-mode="extension">
+        <xpath expr="//*[hasclass('o-mail-Message-contentContainer')]" position="attributes">
+            <attribute name="t-attf-class" add="{{ props.thread.channel_type === 'ai_composer' ? 'flex-column' : '' }}" separator=" "/>
+        </xpath>
+    </t>
+    <t t-inherit="mail.Message.actions" t-inherit-mode="extension">
+        <xpath expr="//button" position="attributes">
+            <attribute name="class" remove="border-0" separator=" "/>
+            <attribute name="t-attf-class" add="{{ action.id === 'insertToComposer' ? 'rounded-3 border-2' : 'border-0' }}" separator=" "/>
+            <attribute name="t-attf-style" add="{{ action.id === 'insertToComposer' ? 'background-color: lightgray; border-color: gray;' : '' }}"/>
+        </xpath>
+        <xpath expr="//button/i" position="replace">
+            <i t-if="action.icon" class="fa-lg" t-att-class="action.icon"/>
+            <span t-if="action.id === 'insertToComposer'">Use this</span>
+        </xpath>
```

Comment:
If you `position="inside"` on button xpath, you don't have to repeat the `<i>`. The inside content is appended to the children.